test/interface.jl

```diff
@@ -847,10 +847,19 @@ end
     for i in 1:length(bs), j in 1:length(bs)
         if i == j
             @test bs[i] == deepcopy(bs[j])
-            @test inv(bs[i]) == inv(deepcopy(bs[j]))
+            @test inverse(bs[i]) == inverse(deepcopy(bs[j]))
         else
             @test bs[i] != bs[j]
         end
     end
 end
 
+@testset "deprecations" begin
+    b = Bijectors.Exp()
+    x = 0.3
+
+    @test let r = forward(b, x)
+        (r.rv, r.logabsdetjac) == with_logabsdet_jacobian(b, x)
+    end
+    @test inv(b) == inverse(b)
```

Comment:
Similarly, here maybe use
```suggestion
    @test @test_deprecated(inv(b)) == inverse(b)
```